.github/workflows/functional_tests.yml

```diff
@@ -7,74 +7,135 @@ on:
     branches: [ master ]
 
 env:
-  DIKTAT_VERSION: 0.1.7
-  KTLINT_VERSION: 0.39.0
   GRADLE_OPTS: -Dorg.gradle.daemon=false  # to speed up gradle run
 
 jobs:
-  run-functional-tests:
-    runs-on: ubuntu-latest
+  run_diktat_from_CLI:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
```

Comment:
Why do we use ubuntu-latest, but previously we were using ubuntu-20.04?

Reply:
It shouldn't matter much, currently `ubuntu-latest` is 18.04, but github has already announced it will be changed to 20.04 soon. I thought that since I introduce other OS here, it will be easier to just use `latest` tag and not manage specific versions for all environments.